packages/logos-docusaurus-theme/src/client/components/mdx/Roadmap/Roadmap.tsx

```diff
@@ -24,6 +24,48 @@ export const Roadmap: React.FC<RoadmapProps> = ({
   children,
   ...props
 }) => {
+  const currentYear = new Date().getFullYear()
+
+  const handleBorderStyle = (year: number, isLast: boolean) => {
+    if (year === currentYear) {
+      return 'solid'
+    } else if (isLast) {
+      return 'none'
+    } else return 'dashed'
+  }
```

Comment:
@jinhojang6, apologies for any confusion, events in the same year should be connected with a solid line, rather than events in the current year.